### DISCUSSION TITLE
2021/01/thinkpadx201i-refresh-3/

### DISCUSSION THREAD

**Discussion by giscus** [Li-Aaron/Li-Aaron.github.io#8] [Announcements]
# 2021/01/thinkpadx201i-refresh-3/

老电脑系列第三篇，给Thinkpad X201i更换屏幕。

https://li-aaron.github.io/2021/01/thinkpadx201i-refresh-3/

**Comment by cancy11** [comment]
请问带蓝牙的屏线型号是什么啊?

**Reply by Li-Aaron** [reply]
呃我那个商家没了，我也看不到型号了。
